src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs

```diff
@@ -204,6 +204,8 @@ public class DurableTaskOptions
         /// </value>
         public bool LogReplayEvents { get; set; }
 
+        public string CustomLifeCycleNotificationHelperType { get; set; }
```

Comment:
Please add documentation comments. Maybe something like "Gets or sets the type name of a custom `<see cref="ILifeCycleNotificationHelper"> to use for handling lifecycle notification events."

Reply:
Thank you for review. Added document comments to public apis.